interfaces/cython/cantera/transport.pyx

```diff
@@ -272,6 +272,11 @@ cdef class DustyGasTransport(Transport):
         def __set__(self, value):
             (<CxxDustyGasTransport*>self.transport).setPermeability(value)
 
+    property thermal_conductivity:
+        def __get__(self):
+            return (<CxxDustyGasTransport*>self.transport).gasTransport().thermalConductivity()
```

Comment:
Here I think we need to document that this is returning the thermal conductivity of the embedded gas object, as opposed to a value specific to the dusty gas model.